### PR TITLE
run ecs markdown-code-format to README.md

### PR DESCRIPTION
**Pull request by samsonasik**
@TomasVotruba here I try run `MarkdownCodeFormatter` at https://github.com/symplify/symplify/pull/2118 to format `README.md`

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -233,17 +225,20 @@ return static function (ContainerConfigurator $containerConfigurator): void {
...
+<?php
+declare(strict_types=1);
+
 class SomeClass
 {
     /**
      * @noRector
      */
-    const NAME = '102';
+    public const NAME = '102';
```
Comment (samsonasik) [inline README.md:236]: here seems on purpose for `@noRector` to exclude from check, so it needs revert.

File: README.md [hunk 2]
```diff
@@ -233,17 +225,20 @@ return static function (ContainerConfigurator $containerConfigurator): void {
...
 
     /**
      * @noRector
      */
-    public function foo()
+    public function foo(): void
```
Comment (samsonasik) [inline README.md:241]: here seems on purpose for `@noRector` to exclude from check, so it needs revert.

**Comment by samsonasik** [comment]
This PR just for demo run MarkdownCodeFormatter against markdown file containing php code https://github.com/symplify/symplify/pull/2118

File: README.md (hunk 2 above)
Comment (TomasVotruba) [reply README.md:241]: It's actually related to Rector, not ECS. Coding standard is correct here

File: README.md (hunk 1 above)
Comment (TomasVotruba) [reply README.md:236]: Also correct here

**Comment by TomasVotruba** [comment]
Thanks for trying it here. Results are great! :+1: 

Could you also prepare Github Action, that does the check? 
Just put it into /inactive-workflows for now, we'll enable it when ECS gets merged.

**Comment by samsonasik** [comment]
yes, I will try

**Comment by TomasVotruba** [comment]
Looks good, thank you :+1: 


I forgot to merge this, so delay is fault on my side

**Pull request merged**